ports/json-dto/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+	"name" : "json-dto",
+	"version-string" : "0.2.11",
+	"homepage" : "https://github.com/Stiffstream/json_dto",
+	"description" : "A small header-only library for converting data between json representation and c++ structs.",
+	"license": "BSD-3-Clause",
+	"dependencies" : [ "rapidjson" ]
+}
```

```suggestion
{
  "name": "json-dto",
  "version-string": "0.2.11",
  "homepage": "https://github.com/Stiffstream/json_dto",
  "description": "A small header-only library for converting data between json representation and c++ structs.",
  "license": "BSD-3-Clause",
  "dependencies": [
    "rapidjson"
  ]
}
```

- The line indent should be two spaces.
- There should be no space before colon in every field.
- There should be one space after colon in every field.
- The dependencies should be placed at separated lines.